docs/gluster-block.8

```diff
@@ -102,20 +103,34 @@ show help message and exit.
 show version info and exit.
 .PP
 
-.SH SUPPORTED JSON-FORMATS
---json | --json-plain | --json-spaced | --json-pretty
+.SH COMMON CLI OPTIONS
+.SS
+[\fBtimeout <seconds>\fR]
+it is the time in seconds that cli can wait for daemon to respond (default: 300)
+.PP
+
+.SS
+[\fB--json*\fR]
+used to request the output result in json format (default: plain text) and supported JSON formats: --json|--json-plain|--json-spaced|--json-pretty
+.PP
 
 .SH EXAMPLES
 .nf
 To create a block device of size 1GiB
 .B # gluster-block create blockVol/sampleBlock ${HOST} 1GiB
 
+To create a block device of size 1GiB with timeout 150 seconds
+.B # gluster-block timeout 150 create blockVol/sampleBlock ${HOST} 1GiB
```

Comment:
@lxbsz 
The default is 300 sec, why would someone want to put timeout less than that :-)
Just an example usage, I understand, I will leave this up to you.

Reply:
@lxbsz I assume, you saw this and thought it is fine to go with 150 :-)